README.rst

```diff
@@ -2,108 +2,260 @@
-  Our package directory is located at: ``src/plone/staticresources/static/components``.
+- **Integrators and Developers:** who want to customize or maintain the front end of a Plone site for their customers:
 
+  - upgrade existing versions or install new ``npm`` packages.
 
-Target audience
----------------
+  - integrate new JavaScript/CSS behavior, eventually wrapped in a ``pattern`` for optimal integration and reuse within
+    Plone front end ecosystem.
 
-This documentation aims at:
+- **Plone Core Developers:** who want to fix Plone bugs or enhance this add-on.
 
-Integrators and Developers
-^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Who want to customize or maintain the front end of a Plone site for their customers:
+How to upgrade the resources in this package
+--------------------------------------------
 
-- upgrade existing versions or install new ``npm`` packages.
+1. Increase ``npm`` package versions in ``package.json``, in sections ``dependencies`` or ``devDependencies``.
 
-- integrate new JavaScript/CSS behavior, eventually wrapped in a ``pattern`` for optimal integration and reuse within Plone front end ecosystem.
+2. Run ``yarn upgrade`` (important: cannot be ``npm``)
 
+3. Run ``./bin/plone-compile-resources -b plone`` or ``./bin/plone-compile-resources -b plone-logged-in`` (whichever
+   bundle you need to re-build).
 
-Themer authors
-^^^^^^^^^^^^^^
+4. Increase the ``last_compilation`` date in ``src/plone/staticresources/profiles/default/registry/bundles.xml``.
 
-Who want to change the look and feel of a Plone site and need to:
+5. Submit a PR and run the tests on Jenkins.
 
-- understand how to correctly bundle changes from existing or new frontend code.
 
+What has changed
+----------------
 
-Plone Core Developers
-^^^^^^^^^^^^^^^^^^^^^
+Between Plone 5.0 and 5.1 these resources were located in ``static/`` directory of ``Products.CMFPlone`` package.
+Starting with Plone 5.2 they're distributed in this independent package due to the specific tooling and workflows used
+to build frontend resources and also the different maintenance and release needs of ``Products.CMFPlone``. For a
+complete list of reasons, read `PLIP 1653 <https://github.com/plone/Products.CMFPlone/issues/1653>`_.
 
-Who want to help enhance this add-on itself.
+.. note::
+  A note on ``yarn``: we require the use of yarn, because it supports installing packages in a custom named directory,
+  which npm does not (even using ``--prefix``, ``node_modules`` is hardcoded). Our package directory is located at:
+  ``src/plone/staticresources/static/components``.
 
+Changes since Plone 5.1.x
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Compiling bundles
------------------
+- The ``toolbar`` pattern from ``Products.CMFPlone.static.toolbar`` has been moved to ``mockup`` package.
+- ``mockup`` package now uses ``npm``'s registry and ``yarn`` instead of ``bower``.
```

Comment:
Remove double backticks around npm, yarn, and bower, and capitalize Yarn. These are not commands, but names of packages.